components/brave_rewards/resources/page/components/monthly_tips_panel.style.ts

```diff
@@ -45,3 +45,9 @@ export const showAll = styled.div`
     color: #4C54D2;
   }
 `
+
+export const nextContributionDate = styled.div`
+  font-size: 12px;
+  line-height: 18px;
+  color: #858999;
```

Comment:
We have some old/outdated color vars on the Rewards page, but there's not a whole lot to gain from using them. When we start to bring in the leo colors, this will change!